src/environment.js

```diff
@@ -73,6 +73,10 @@ export const sortAppsPair = (app1, app2) => {
 const appOverrides = {
   // Needed to change app name on sidebar for old versions whose aragonPM repo content cannot be changed anymore
   [appIds['TokenManager']]: { name: 'Tokens' },
+  [appIds['Agent']]: {
+    start_url: '/index.html',
+    script: 'script.js',
+  },
```

Comment:
Isn’t this being read from the manifest already? https://github.com/aragon/aragon-apps/pull/943/files#diff-97b8442de4c498bf877db0e100798b5b

Reply:
I assumed so, but in my tests it was not working without this, maybe I'm missing something in the manifest.

Reply:
It's because we haven't updated the Agent app's content on aragonPM yet. We won't need this override once we do; let's merge this after we do a publish?

Reply:
That makes sense, thanks for clarifying Brett, in the meantime this can be used to test out the new UI app.

Reply:
We'll be deploying the Agent app with an updated manifest soon, so I've removed the overrides in f6be94f.